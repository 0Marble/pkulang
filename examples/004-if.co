fn main() void {
	let x: int = 10;
	let y: int = 20;

	% {} around the branches is optional
	% unless the branch is an if/while/... (can itself have a body)
	if (x < 30) println("ok");

	if (x < y) {
		% print just prints all given arguments
		println(x, " < ", y);
	} else {
		println(y, " < ", x);
	}
}

