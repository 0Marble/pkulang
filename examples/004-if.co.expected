ok
10 < 20
